src/components/GalaxyModal.js

```diff
@@ -45,7 +45,7 @@ export const GalaxyModalBase = Utils.withDisplayName('GalaxyModal')(
     const [loading, setLoading] = useState(false)
     const [shouldDeleteDisk, setShouldDeleteDisk] = useState(false)
 
-    const currentDataDisk = currentPersistentDisk(apps, galaxyDataDisks)
+    const currentDataDisk = currentPersistentDisk(apps, galaxyDataDisks, workspace.workspace.name)
```

Comment:
Given the deconstruction

```javascript
workspace: { workspace: { namespace, bucketName, name: workspaceName, googleProject } }
```

in `GalaxyModalBase` input definition

```javascript
export const GalaxyModalBase = Utils.withDisplayName('GalaxyModal')(
  ({
    onDismiss, onSuccess, apps, galaxyDataDisks, workspace, workspace: { workspace: { namespace, bucketName, name: workspaceName, googleProject } },
    isAnalysisMode = false
  }) => {…}
```

don't we already have `workspace.workspace.name`?

```suggestion
    const currentDataDisk = currentPersistentDisk(apps, galaxyDataDisks, workspaceName)
```